common/log/interface.go

```diff
@@ -44,6 +44,8 @@ type (
 		Info(msg string, tags ...tag.Tag)
 		Warn(msg string, tags ...tag.Tag)
 		Error(msg string, tags ...tag.Tag)
+		DPanic(msg string, tags ...tag.Tag)
+		Panic(msg string, tags ...tag.Tag)
```

Comment:
I have to point out that this is a breaking change for anyone who has custom logger and provide it using `serverOptions`. I believe we can afford it but need to mention  on 1.20 release notes. I am tagging this PR with 1.20 tag.

Reply:
fortunately it will break at compile time